crucible-saw/src/Lang/Crucible/Backend/SAWCore.hs

```diff
@@ -1007,6 +1007,25 @@ evaluateExpr sym sc cache = f []
                SAWExpr <$> SC.scArrayUpdate sc sc_idx_type sc_elm_type sc_arr sc_idx sc_elm
           | otherwise -> unimplemented "multidimensional UpdateArray"
 
+        B.CopyArray w a_repr dest_arr dest_idx src_arr src_idx len ->
```

Comment:
Why are we not handling the "equal array range" constructor? Did I miss something?

Reply:
On second thought, nevermind.  This is basically dead code now.  I should probably go ahead and delete it.